spec/Section 7 -- Response.md

```diff
@@ -89,37 +108,25 @@ found at `["hero", "friends"]`, the hero's first friend at
 
 ### Data
 
-The {"data"} entry in the response will be the result of the execution of the
-requested operation. If the operation was a query, this output will be an object
-of the query root operation type; if the operation was a mutation, this output
-will be an object of the mutation root operation type.
+The {"data"} entry in the _execution result_ will be the result of the execution
+of the requested operation. If the operation was a query, this output will be an
+object of the query root operation type; if the operation was a mutation, this
+output will be an object of the mutation root operation type.
 
 The response data is the result of accumulating the resolved result of all
 response positions during execution.
 
-If an error was raised before execution begins, the {"data"} entry should not be
-present in the response.
+If an error was raised before execution begins, the _response_ must be a
+_request error result_. The {"data"} entry must not be present in this map.
```

Comment:
Feels to me like the normative rule on {"data"} not being present in a request error result should live in the request error result section.

```suggestion
If an error was raised before execution begins, the _response_ must be a
_request error result_ which will result in no response data.
```